m4_sessions/1803-inning/group_1.md

```diff
@@ -1,13 +1,23 @@
 Group 1
 
-Group Members:
+Group Members: James Logue, Melissa Wagasky, Melena Suliteanu
 
-Name of Session:
+Name of Session: Your Smartphone is a Civil Rights Issue: Digital Privacy as a Developer
 
 Session Description (Why should students come to this session):
+If you care about digital privacy and your role in contributing to good practices.
 
 Session Outcomes (What will students learn/leave with?):
+You will walk away with an understanding of the current landscape of digital privacy, the companies in the privacy ecosystem, and a manifesto around your own beliefs of privacy as a developer.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
-
-Submit by the end of Gear Up on Friday and tag @emhickmann
+We'll cofacilitate equally.
+Intro
+Ted Talk/video
+Small discussion about video
+From minute 15: https://www.ted.com/talks/glenn_greenwald_why_privacy_matters/discussion?language=en#t-475349
+(important: complacency and conformist behavior when being watched)
+Breakout to research privacy policies of different products. (We will provide a diversity of products)
+Group discussion around privacy policies people researched.
+Discuss how to think about privacy as a developer
+Craft personal manifesto of beliefs from session.
```

Comment:
Love that you are doing this!  This will be excellent!  Leave at least 10 minutes for this and potentially time to share out.  Maybe even start a slack channel and have people post their manifesto!  Ideal if you each have one already crafted.